core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -1,6 +1,10 @@
 default: api.ci
 nonKubernetes: api.ci
 groups:
+  "app.ci":
+    jobs:
+    #name: app.ci-jobs
```

Comment:
stale comment?

Reply:
I want to keep it there.
For each group, we leave a comment `#name: ...` to remind me of what the group is about.
We had `name` as a field of the `json` but it did not go through review. :)

Reply:
I don't understand. On line four, there is the name `'app.ci"`. You are afraid you forgot about that when you get to line six so you add a comment there?